CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 -->
 
 ## [Unreleased]
+- Added missing Norwegian (nb-NO) translations, by [@taarskog](https://github.com/taarskog)
```

Comment:
Please also add your github name to the `LOCALIZATION.md` file. :)

Reply:
Not sure what you want me to do. My name is already in there  as @taarskog under [nb-no](https://github.com/taarskog/BotFramework-WebChat/blob/master/LOCALIZATION.md).